docs/index.rst

```diff
@@ -34,6 +34,8 @@ Main Features
   installation/index
```

Comment:
I have issued a PR to adjust the document hierarchy below. Please review it.
- #415